Content.Server/GameObjects/Components/Fluids/RagComponent.cs

```diff
@@ -0,0 +1,132 @@
+﻿#nullable enable
+using Content.Server.GameObjects.Components.Chemistry;
+using Content.Shared.Chemistry;
+using Content.Shared.Interfaces;
+using Content.Shared.Interfaces.GameObjects.Components;
+using Content.Shared.Utility;
+using Robust.Server.GameObjects.EntitySystems;
+using Robust.Shared.GameObjects;
+using Robust.Shared.GameObjects.Systems;
+using Robust.Shared.Localization;
+using Robust.Shared.Serialization;
+using System.Threading.Tasks;
+
+namespace Content.Server.GameObjects.Components.Fluids
+{
+    /// <summary>
+    /// For cleaning up puddles
+    /// </summary>
+    [RegisterComponent]
+    public class RagComponent : Component, IAfterInteract
```

Comment:
Is rag supposed to be in this PR?

Reply:
No actually, I believe it got in there by mistake. I'll commit the changes.

Reply:
I'm happy to merge if it's removed but I won't be able to do it for a bit.